features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -78,15 +83,28 @@ internal class Base64Serializer private constructor(
     // region private
 
     @WorkerThread
-    private fun serialiseBitmap(
+    private fun serializeBitmap(
         drawable: Drawable,
+        displayMetrics: DisplayMetrics,
         bitmap: Bitmap,
         shouldCacheBitmap: Boolean,
         imageWireframe: MobileSegment.Wireframe.ImageWireframe,
-        base64SerializerCallback: Base64SerializerCallback?
+        base64SerializerCallback: Base64SerializerCallback,
+        isFailover: Boolean
```

Comment:
should we rename this property or add any explanation comment at least? Currently it is not clear what it is doing tbh.

Reply:
yes, there is a comment in convertBmpToBase64, but I'll try to make this clearer